02 - Community Expansions/02.01 Plugins by Category/Plugins to export and import markdown files.md

```diff
@@ -7,9 +7,9 @@ publish: true
 ---
 
 
-# Convert
+# Plugins to export and import into Markdown
```

Comment:
I think that category is rather ambiguous, since it can mean two quite different things:
1) Plugins that convert *content* into markdown content (e.g. Tweets to Markdown)
2) Plugins that convert a *file* into .md (e.g. a PDF into Markdown)

Reply:
I think I'd make two plugin categories out of that

Reply:
Opened #62 for this